screens/settings/settings.py

```
@@ -39,6 +51,9 @@ def on_setting_press(self, setting_name):
     
     def navigate_back(self):
```

Ruff (0.11.9)

Redefinition of unused `navigate_back` from line 18

(F811)